app/templates/home.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div class="container col-sm-12">
+        <div class=" col-sm-8">
+            <strong>
+                <p class="home-page-header">Welcome to NYC Government’s home for filing Freedom of Information Law
```

Comment:
If you're going to use a class, it is a better idea to include `font-weight: bold` in the css rather than resorting to using the `strong` tag. Also, there is an unnecessary  amount of spacing in between lines, why not just use an `h1` (or whatever size is best) tag instead of a styled paragraph? If the header tag doesn't do the job, can you find a way of reducing the spacing, maybe use `line-height`?